src/wallet/rpcwallet.cpp

```diff
@@ -1352,10 +1356,12 @@ UniValue listtransactions(const JSONRPCRequest& request)
 
     if (request.fHelp || request.params.size() > 4)
         throw std::runtime_error(
-            "listtransactions (dummy count skip include_watchonly)\n"
+            "listtransactions ( label count skip include_watchonly )\n"
```

Comment:
From the documentation below, label is a string, so should be surrounded by quotation marks in the oneline doc. 

```suggestion
            "listtransactions ( \"label\" count skip include_watchonly )\n"
```